webapi/middleware.go

```diff
@@ -305,58 +305,77 @@ func vspAuth() gin.HandlerFunc {
 			return
 		}
 
-		// If the ticket was found in the database, we already know its
-		// commitment address. Otherwise we need to get it from the chain.
-		var commitmentAddress string
-		if ticketFound {
-			commitmentAddress = ticket.CommitmentAddress
-		} else {
-			dcrdClient := c.MustGet("DcrdClient").(*rpc.DcrdRPC)
-
-			resp, err := dcrdClient.GetRawTransaction(hash)
+		// validate validates the signature in the request as signing
+		// the request body with private key corresponding to sAddr.
+		// fatal will cause the client to be notified of failure if
```

Comment:
s / fatal will cause ... / Verification failure results in the client getting notified of the cause.

s / Returns wether verification succeeded and sets some header values if success. / Returns whether verification succeeded.

Reply:
ok